tokio/src/runtime/driver.rs

```diff
@@ -249,7 +240,6 @@ cfg_time! {
     pub(crate) enum TimeDriver {
         Enabled {
             driver: crate::runtime::time::Driver,
-            handle: crate::runtime::time::Handle,
```

Comment:
This is the additional time handle ref to remove. Removing this gets rid of arc clones below.